scripts/autoscaling/init_multicloud.sh

```diff
@@ -250,6 +250,12 @@ elif [[ $cloud == *"Microsoft"* ]]; then
         _CLOUD_INSTANCE_IMAGE_ID=""
     fi
     _CLOUD_PROVIDER=AZURE
+
+    CHECK_AZURE_EVENTS_COMMAND="curl -k -H Metadata:true http://169.254.169.254/metadata/scheduledevents?api-version=2017-11-01 2> /dev/null | grep Preempt && kubectl label node $(hostname) preempted=true --kubeconfig='/etc/kubernetes/kubelet.conf'"
+
+    # run 3 cron jobs to be able to check events each 20 seconds
+    crontab -l | { cat ; echo -e "* * * * * $CHECK_AZURE_EVENTS_COMMAND \n* * * * * sleep 20 && $CHECK_AZURE_EVENTS_COMMAND \n* * * * * sleep 40 && $CHECK_AZURE_EVENTS_COMMAND" ; } | crontab -
```

Comment:
I was thinking about this a bit more - why not to use a simple shell script with an infinite loop and a `sleep` instead of the complex `cron` configuration?
@SilinPavel do you foresee any issues with that?

Reply:
@sidoruka I just think that it would 100% avoid any problem between iteration run. And each time cron run will be independent of previous one.